tests/lsstseries_tests/test_ensemble.py

```diff
@@ -5,7 +5,19 @@
 from lsstseries.analysis.stetsonj import calc_stetson_J
 from lsstseries.analysis.structurefunction2 import calc_sf2
 
+
 # pylint: disable=protected-access
+def test_with():
+    """Test that we open and close a client on enter and exit."""
+    with ensemble() as ens:
```

Comment:
I know this isn't from this pull request, but can we change the class name to be capitalized?

I'm happy to make a different issue and do it myself if we don't want to here.

Reply:
Ah, yes, good eye @smcguire-cmu . @delucchi-cmu , up to you if you want to take this on here or just have this be it's own thing

Reply:
Happy to do in another PR. I was getting yelled at a lot by pylint about this one, but I was really trying not to step on too many toes!